pyscf/neo/ase.py

```diff
@@ -86,52 +101,85 @@ def calculate(self, atoms=None, properties=['energy', 'forces', 'dipole'],
                   nuc_basis=self.parameters.nuc_basis,
                   charge=self.parameters.charge,
                   spin=self.parameters.spin)
-        if self.parameters.spin == 0:
-            mf = neo.CDFT(mol, epc=self.parameters.epc)
+
+        # If there is no mf, create one
+        if self.mf is None:
+            if self.parameters.spin == 0:
+                mf = neo.CDFT(mol, epc=self.parameters.epc)
+            else:
+                mf = neo.CDFT(mol, unrestricted = True)
+            mf.mf_elec.xc = self.parameters.xc
+            if self.parameters.atom_grid is not None:
+                mf.mf_elec.grids.atom_grid = self.parameters.atom_grid
+            if self.parameters.add_vv10:
+                mf.mf_elec.nlc = 'VV10'
+                mf.mf_elec.grids.prune = None
+                mf.mf_elec.nlcgrids.atom_grid = (50,194)
+                mf.mf_elec.nlcgrids.prune = dft.gen_grid.sg1_prune
+            if isinstance(self.parameters.init_guess, str):
+                if self.parameters.init_guess != 'last_step' and self.parameters.init_guess != 'elec_only':
```

Comment:
Too long line. Break the line.

Reply:
@zheliu17 Here is the correct check you did for this block, but you forgot for the `else` block.